crates/bevy_asset/src/lib.rs

```diff
@@ -64,7 +64,9 @@ pub enum AssetStage {
 #[derive(Default)]
 pub struct AssetPlugin;
 
-/// [`AssetServer`] settings.
+/// Settings for the [`AssetServer`].
+///
+/// This resource must be added before the [`AssetPlugin`] to take effect.
```

Comment:
Since most people use the DefaulPlugins it's probably important to mention it directly
```suggestion
/// This resource must be added before the [`AssetPlugin`] or [`DefaultPlugins`] to take effect.
```

Reply:
I don't think I can link to the `DefaultPlugins` struct as it doesn't reside in a dependency of `bevy_asset`.

Reply:
yeah, it's a bit of an unfortunate limitation.